ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -38,9 +46,10 @@
   (get-bulk-update-task-status
     [db task-id]
     ;; Returns a status for the particular task
-    (su/find-one db (su/select [:status :status-message]
-                     (su/from "bulk_update_task_status")
-                     (su/where `(= :task-id ~task-id)))))
+    (let [status (su/find-one db (su/select [:status :status-message]
+                                   (su/from "bulk_update_task_status")
+                                   (su/where `(= :task-id ~task-id))))]
+      (util/map-keys->kebab-case status)))
```

Comment:
I tend to avoid `let`s with a single-use variable. The threading macros are usually clear enough to be just as self-documenting (and the function name + docstring can make up for any lack there):

```clj
(-> db
    (su/find-one (su/select [:status ...]
                            (su/from ...)
                            (su/where ...)))
    (util/map-keys->kebab-case))
```

Using `->>` here probably wouldn't be a good idea, since that would mean breaking up your select statement in an artificial manner ...